apps/opik-backend/src/main/resources/liquibase/db-app-state/migrations/000002_create_usage_usage_information_table.sql

```diff
@@ -0,0 +1,19 @@
+--liquibase formatted sql
+--changeset thiagohora:create_usage_usage_information_table
+
+CREATE TABLE metadata (
```

Comment:
This table is useful as a generic metadata storage, in this case the anonymous_id, which is an UUID. But it isn't strictly needed for the usage_information functionality.

The pro is that we would be able to reuse the `anonymous_id`, although there's no use case for that.
The con is that the schema isn't specific for this use case.

I'm not against this table, but we need to check how it fits with this whole change and the future features in this service.

Reply:
I agree; I thought about it in case we need to collect other data regarding the client's installation. That's why I tried this more generic format.